### ISSUE TITLE
Android 15 QPR2 Beta 2 breaks Wavelet (on Pixel 8 Pro at least)

### ISSUE THREAD

**Issue by trip6kitty** [Pittvandewitt/Wavelet#330]
It appears with the latest QPR2 Beta 2, Wavelet no longer works.  Just posting this here as an FYI as it appears something has changed on Android's side that is preventing Wavelet from working correctly.  

I posted on reddit and it appears another person is able to recreate the issue.  

Edit:

I tested against a Pixel 4 on Android 14 (via LineageOS). Used multipoint on my Nothing Ear (2) and hooked it up to both my Pixel 8 Pro and the Pixel 4. Both using Wavelet, I noticed the Pixel 8 Pro is missing the Reverberation option. 

I do notice a difference in audio quality. The P4 sounds a lot better while the P8P sounds bad.

**Comment by 7RST1** [comment]
Same device, same issue, over here with my Bose QC 45s

**Comment by trip6kitty** [comment]
> Same device, same issue, over here with my Bose QC 45s
> 
> EDIT: Issue is not present on **23.03**

**Comment by trip6kitty** [comment]
Closed on accident -

I will try with an older version 23.03

**Comment by Pittvandewitt** [comment]
Is any of you able to take a logcat (`adb logcat`) while disabling and enabling Wavelet via the power button?

**Comment by trip6kitty** [comment]
Hi @Pittvandewitt , I have the logcat file here.  This was my first time capturing a logcat file so I hope I did it correctly.

This is what I did:

1. Run `adb logcat`
2. Unlock my Pixel 8 Pro (music is already playing).
3. Enable Wavelet for a couple seconds.
4. Disable Wavelet for a couple seconds.
5. End logcat capture via CTRL+C.
6. Save logcat file via `adb logcat -d > logcat.txt`

Let me know if I did it incorrectly, and I could try again.

[logcat.txt](https://github.com/user-attachments/files/18427903/logcat.txt)

**Comment by trip6kitty** [comment]
Thank you so much. I have starred the issue on the tracker.

**Comment by trip6kitty** [comment]
FYI the bug tracker is requesting more info or the bug will be closed per the last Google comment.

**Comment by andreasoppliger** [comment]
> Looks like this is an issue on Pixel 8 as of the March update now that Android 15 QPR2 is out of beta and officially released.

Can confirm that. Build-Nr. BP1A.250305.019 broke it on my Pixel 8 too.

**Comment by DangerNdle** [comment]
Also on Build-Nr. BP1A.250305.019, wanted to pull my hair out trying to figure the issue out only to find this issue thread. Just the reverb effects aren't working for me since the update.

**Comment by DangerNdle** [comment]
Do you mean there was an update that fixes it via wavelet or android?

**Comment by trip6kitty** [comment]
@DangerNdle you need version 25.03. Maybe APK mirror will have it if you're not getting an update via play store.

**Comment by DangerNdle** [comment]
@trip6kitty Will try that, thanks!

**Comment by andreasoppliger** [comment]
> Sorry to seem confused; where do I grab the update from? I don't see it anywhere and the Google Play store hasn't shown a new update for me yet.

AIDL mode i working for me as well, but had to finetune the eq a bit for same sound as before. I'm in beta mode on Google Play, maybe this is an option for you as well.

**Comment by DangerNdle** [comment]
In case it helps anyone else that isn't seeing the update, I cleared my data on the Google Play app and restarted my phone. After doing so, the update showed up. Just restarting without clearing anything wouldn't work, so figured I'd leave this here for anyone else with the issue.

**Comment by illumiN8i** [comment]
@andreasoppliger I agree, it does sound a bit different from before with the new AIDL mode. The AutoEq effect seems weaker now. I hear less difference when toggling it on and off. I can't say if it's more or less accurate, but it does sound different to me.
